### PR TITLE
Feature/add dotenv

### PR DESCRIPTION
**Pull request by Walter-Dalla**
ADD:
It is now easier to configure the env variables, just put them in .env!

image

This change was documented in readme.md as:

image

FIXED:
If your geocode token were invalid, it would generate an undefined error:

image

Now it will print the error message to the user, like this:

image

### CODE REVIEW

**Comment by Walter-Dalla** [comment]
#hacktoberfest

**Comment by jmtellez** [comment]
@Walter-Dalla  can you run `npm link` on root directory and then from any other path run `cli-mate <location>`
I tried using dotenv module but when creating symlink to project it cannot find it.

**Comment by CarlyRaeJepsenStan** [comment]
Thanks - this should be merged ASAP - I thought those errors were on my machine haha.

**Comment by jmtellez** [comment]
error will occur if you dont have the env vars

File: utils/forecast.js [hunk 1]
```diff
@@ -2,12 +2,12 @@ const request = require("postman-request");
 const ora = require("ora");
 
 const forecast = (lat, lon, units = "f", callback) => {
-  const weatherstackURI = `http://api.weatherstack.com/current?access_key=${process.env.WEATHERSTACK}&query=${lat},${lon}&units=${units}`;
+  const weatherStackURI = `${WEATHER_STACK_URL}${WEATHER_STACK_PATH}?access_key=${process.env.WEATHERSTACK}&query=${lat},${lon}&units=${units}`;
```
Comment (jmtellez) [inline utils/forecast.js:5]: `${WEATHER_STACK_URL}${WEATHER_STACK_PATH}` not defined

**Comment by jmtellez** [comment]
this project does not require `dotenv` as its meant to be installed through npm. 
`dotenv` would only work for contributors when testing

**Pull request closed without merging**